examples/webhook-signing/express-ts.ts

```diff
@@ -0,0 +1,89 @@
+import Stripe from 'stripe';
+import express from 'express';
+import bodyParser from 'body-parser';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const stripe: Stripe = new Stripe(process.env.STRIPE_API_KEY!, {
+  apiVersion: '2019-12-03',
+  typescript: true,
+});
+
+/**
+ * You'll need to make sure this is externally accessible. ngrok (https://ngrok.com/)
+ * makes this really easy.
+ *
+ * Alternatively, you could use the stripe-cli in forward mode: https://github.com/stripe/stripe-cli
+ *
+ * To run this file, just provide your Secret API Key and Webhook Secret in a .env file in this directory like so:
+ *
+ * STRIPE_API_KEY=sk_test_XXX
+ * WEBHOOK_SECRET=whsec_XXX
+ *
+ * Then run "npm run tsc", which will convert this TypeScript file to JS and then run it.
+ *
+ * For use with the stripe-cli, run the following:
+ *
+ * 1. "stripe listen --forward-to localhost:3000/webhooks"
+ * 2. Copy the provided webhook signing secret to your .env file
+ * 3. In a new terminal window: "npm run tsc"
+ * 4. In yet another new terminal window: "stripe trigger payment_intents.succeeded"
+ */
+
+const webhookSecret: string = process.env.WEBHOOK_SECRET!;
+const app: express.Application = express();
+
+// Only use the raw body parser for webhooks
+app.use(
+  (
+    req: express.Request,
+    res: express.Response,
+    next: express.NextFunction
+  ): void => {
+    if (req.originalUrl === '/webhooks') {
```

Comment:
Is this our best-practice? Does line 55 not work here?

Reply:
Line 55 does work, but I've seen people get tripped up by how express' middleware works. Having this block here will hopefully nip that confusion in the bud (and makes it so all other routes use the json body parser).